client/src/components/layout/ReviewTile.js

```diff
@@ -1,12 +1,16 @@
 import React from "react";
+import ReviewEditButton from "./ReviewEditButton";
+import ReviewDeleteButton from "./ReviewDeleteButton"
 
-const ReviewTile = ({review}) => {
+const ReviewTile = ({ review, user, getPlanet }) => {
 
   return(
-    <>
+    <div className = "review">
     <li>{review.body}</li>
     <li>Rating: {review.rating}</li>
-    </>
+    { user ? <ReviewEditButton review={review} user={user} /> : null }
+    { user ? <ReviewDeleteButton review={review} user={user} getPlanet={getPlanet} /> : null }
+    </div>
```

Comment:
Im not seeing any indentation in this div. Make sure child elements are indented in